nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -43,6 +43,20 @@ let
       cd $TMPDIR
```

Comment:
Minor comment: the comment above, "Create a directory for exchanging data..." now seems out-dated because the directory is also used for the disk image.